services/secondary-storage/server/src/main/java/org/apache/cloudstack/storage/resource/NfsSecondaryStorageResource.java

```diff
@@ -1137,11 +1167,11 @@ protected File downloadFromUrlToNfs(String url, NfsTO nfs, String path, String n
             try (FileOutputStream outputStream = new FileOutputStream(destFile);) {
                 entity.writeTo(outputStream);
             } catch (IOException e) {
-                s_logger.debug("downloadFromUrlToNfs:Exception:" + e.getMessage(), e);
+                s_logger.error("downloadFromUrlToNfs:Exception:" + e.getMessage(), e);
```

Comment:
Suggestion:

We could improve logging here by adding some data, like:

```java
s_logger.error(String.format("Unable to write downloaded data from [%s] to [%s] due to [%s].", url, destFile, e.getMessage()), e);
```

Reply:
thanks will do it.